#def set_foo():
 #   foo = 'bar'

#set_foo()
#print(foo)

#What happens when you run the code above?

print("error occurs, because variable foo defined locally within the function")

